<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ParentWindow</class>
 <widget class="QMainWindow" name="ParentWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DataViz 0.0</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QMdiArea" name="WindowsManager">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>120</y>
      <width>200</width>
      <height>160</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionLoad_Dataset"/>
   </widget>
   <widget class="QMenu" name="menuPlot">
    <property name="title">
     <string>Plot</string>
    </property>
    <addaction name="action1D_Plot"/>
    <addaction name="actionHist_Plot"/>
   </widget>
   <widget class="QMenu" name="menuAnalysis">
    <property name="title">
     <string>Analysis</string>
    </property>
    <addaction name="actionFunction"/>
    <addaction name="actionInterpolation"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionHelp"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuPlot"/>
   <addaction name="menuAnalysis"/>
   <addaction name="menuHelp"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionLoad_Dataset"/>
   <addaction name="separator"/>
   <addaction name="action1D_Plot"/>
   <addaction name="actionHist_Plot"/>
   <addaction name="separator"/>
   <addaction name="actionFunction"/>
   <addaction name="actionInterpolation"/>
   <addaction name="separator"/>
   <addaction name="actionHelp"/>
   <addaction name="actionAbout"/>
  </widget>
  <action name="actionLoad_Dataset">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/dataset.svg</normaloff>:/icons/dataset.svg</iconset>
   </property>
   <property name="text">
    <string>Load Dataset</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/aboutSymbol.svg</normaloff>:/icons/aboutSymbol.svg</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/helpSymbol.svg</normaloff>:/icons/helpSymbol.svg</iconset>
   </property>
   <property name="text">
    <string>Help</string>
   </property>
  </action>
  <action name="actionHist_Plot">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/histogram.svg</normaloff>:/icons/histogram.svg</iconset>
   </property>
   <property name="text">
    <string>Histogram Plot</string>
   </property>
  </action>
  <action name="actionFunction">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/function.svg</normaloff>:/icons/function.svg</iconset>
   </property>
   <property name="text">
    <string>Function</string>
   </property>
  </action>
  <action name="actionInterpolation">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/Interpolation.svg</normaloff>:/icons/Interpolation.svg</iconset>
   </property>
   <property name="text">
    <string>Interpolation</string>
   </property>
  </action>
  <action name="action1D_Plot">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/graph.svg</normaloff>:/icons/graph.svg</iconset>
   </property>
   <property name="text">
    <string>1D Plot</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
